### DISCUSSION TITLE
Welcome to solid-start Discussions!

### DISCUSSION THREAD

**Discussion by nksaraf** [solidjs/solid-start#146] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Jutanium** [comment]
Hey, I'm Dan!
I'll be working with @nksaraf to help document SolidStart.
If this is something you'd be interested in helping with, please let us know!

**Reply by Zachiah** [reply]
I would definitely be interested in helping! I have already made significant commits on the docs
